openmmapi/include/openmm/ATMForce.h

```diff
@@ -101,35 +101,46 @@ class OPENMM_EXPORT ATMForce : public OpenMM::Force {
      * Normally, all of the particles in the OpenMM's System should be added to the ATMForce
      * in the same order as they appear in the System.
      *
-     * @param particle    the index of the particle
-     * @param dx          the x component of the displacement vector in nm
-     * @param dy          the y component of the displacement vector in nm
-     * @param dz          the z component of the displacement vector in nm
+     * @param displacement  the displacement vector in nm
      * @return the index of the particle that was added
+     *
+     * The displacement vector can have 3 or 6 elements. A displacement with 3 elements
+     * specifies the x, y, z coordinates of the displacement for the target state and 
+     * the particle is not displaced in the initial state (most common).
+     *
+     * If the displacement has 6 elements, the first 3 specify the x, y, z coordinates of 
+     * the displacement for the target state as above, and the last 3 elements specify the coordinates 
+     * of the displacement for the initial state.
      */
-    int addParticle(int particle, double dx, double dy, double dz);
+    int addParticle(const std::vector<double>& displacement=std::vector<double>());
```

Comment:
If we want to make it a single vector instead of three numbers, let's use a Vec3.  That will be much cleaner.  A `vector<double>` can have any number of components.

Reply:
Yes, that's the intent. No problem. It can wait.